src/app/utils.js

```diff
@@ -230,7 +230,22 @@ export const debounce = (fn, wait) => {
   };
 };
 
+/**
+  Returns a bool on whether a user can admin controllers
+  @param {Obj} conn The user connection.
+  @returns {Bool} True or false
```

Comment:
I think the type is `Boolean`
```suggestion
  @returns {Boolean} If they are an admin or not.
```